tests/yam/agama/agama_installation.pm

```diff
@@ -0,0 +1,35 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run interactive installation with Agama,
+# run playwright tests directly from the Live ISO.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+
+use testapi qw(
+  assert_screen
+  assert_script_run
+  select_console
+  script_output
+  upload_logs
+);
+use power_action_utils 'power_action';
+
+sub run {
+    my ($self) = @_;
+
+    assert_screen('agama_product_selection', 120);
+    $testapi::password = 'linux';
+    select_console 'root-console';
+
+    assert_script_run('playwright test --trace on --project chromium --config /usr/share/agama-playwright take_screenshots', timeout => 600);
+
+    upload_logs(script_output('find . -name trace.zip'));
```

Comment:
I'm not sure about that, it could be more than one result...
it should be in a fixed folder in the SUT, could you please provide the path so we could think of other solutions?